### DISCUSSION TITLE
New to Github (Beginner)

### DISCUSSION THREAD

**Discussion by wikiahmd** [community/community#110749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I just got introduced to Github through Coursera by doing an online course on Mobile App Development. We learnt about Version Control and everything was going well until I came to Github which kept me lagging. I am having a really hard time performing exercises on Github that Coursera has given me and I'm tryin to use the best of the resources I've got so I can study Github commands but I'm still facing difficulty. I know what Version Control and remote repositories are but I have a hard time with the commands. Please help

**Comment by byteakp** [comment]
I have also started learning the git command and to work efficiently on my project its great ...Youtube #110749

**Comment by thearyansingh** [comment]
Try to constribute daily and also  make sure tto show up  your daily progress on github by pushing your projects

**Comment by SebastienXia** [comment]
Hello, @wiqar-gith 
If you have difficulties, there is always the Github documentation for that : https://docs.github.com/en
